### PR TITLE
Update: Examples for Terra-Popup

### PR DESCRIPTION
**Pull request by SG067724**
Jira Link-UXPLATFORM-8504
Description - Updated the examples for terra-pop with content and interactive elements.

### CODE REVIEW

**Comment by SG067724** [comment]
Screenshot 2023-03-28 at 1 12 51 PM
Screenshot 2023-03-28 at 1 12 57 PM
Screenshot 2023-03-28 at 1 13 03 PM

**Comment by SG067724** [comment]
> @SG067724 , do we need this examples updates ? does the example that exists currently for popup does not satisfy here ?

The current examples do not have any content. All the popup examples have just placeholder.

**Comment by ShettyAkarsh** [comment]
@SG067724  please provide the proper description on the PR. regarding the changes that is done.

**Comment by ShettyAkarsh** [comment]
@SG067724  let keep the example more related to the real time scenario. for example the show details button that is added.
the button used to open the popup and the content showed in the popup is not related.  

lets update the button text shown to something like this.

"Show more menu option"

and once the popup is open. use these pills component to have a interactive elements with following option

piil 1: - Menu Option 1
pill 2:- Menu option 2

this way you can cover the interactive element scenario as well

**Comment by ShettyAkarsh** [comment]
> > @SG067724 please provide the proper description on the PR. regarding the changes that is done.
> 
> Updated.

i still do not see the title updated properly.

File: packages/terra-framework-docs/CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,9 @@
 
 ## Unreleased
 
+* Added
+  * Examples for Popup
+
 * Added
   * Added package-lock.json
```
Comment (sdadn) [inline packages/terra-framework-docs/CHANGELOG.md:9]: These can be under the same section:
```suggestion
* Added
  * Examples for Popup
  * Added package-lock.json
```

**Comment by supreethmr** [comment]
@SG067724 remove Jira link and just provide Jira Key on PR description As we are not supposed to add internal links on open source forums.

File: packages/terra-framework-docs/CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,9 @@
 
 ## Unreleased
 
+* Added
+  * Examples for Popup
```
Comment (supreethmr) [inline packages/terra-framework-docs/CHANGELOG.md:6]:
```suggestion
  * Added examples for Popup
```

**Pull request closed without merging**